scripts/image/dither.m

```diff
@@ -0,0 +1,379 @@
+## Copyright (C) 2025 Leonardo Araujo <leolca@gmail.com>
+##
+## This program is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
```

Comment:
The "standard" copyright statement in other files has two spaces after the full-stop in this line.